.github/workflows/linters.yml

```diff
@@ -74,7 +74,7 @@ jobs:
           restore-keys: ${{ runner.os }}-node-
       - run: npx pnpm install
       - run: npm run lint:ci
-      - run: npm run lint:ci -- --format junit -o reports/junit/eslint-results.xml
+      - run: npm run lint:ci -- --format junit -o reports/junit/eslint-results.xml --max-warnings=0
```

Comment:
Do we need to run `lint:ci` twice? @Jack-Works  @septs

Reply:
We need CI provide warnings

Reply:
<img width="924" alt="CleanShot 2021-06-25 at 14 45 36@2x" src="https://user-images.githubusercontent.com/3842474/123382028-076fb600-d5c4-11eb-94ec-0fb4c0c5d9ca.png">
<img width="709" alt="CleanShot 2021-06-25 at 14 45 51@2x" src="https://user-images.githubusercontent.com/3842474/123382047-0b9bd380-d5c4-11eb-93d7-7af97bf671d5.png">